### ISSUE TITLE
Extract-AppFileToFolder requires absolute paths

### ISSUE THREAD

**Issue by jhoek** [microsoft/navcontainerhelper#1577]
**Describe the issue**
I found this when I passed a relative path ("./Foo.app") to the -app parameter of `Run-ALValidation`. `Extract-AppFileToFolder` uses a .NET file stream to read an app file. The file stream class is not aware of/does not respect the active working directory in PowerShell. I haven't tried, but I assume it will also trip over PowerShell-specific paths, such as "~/Foo.app" and "MyCustomDrive:/Foo.app". I could not find any code that proactively tries to resolve the PowerShell path to something that .NET can understand?

**Scripts used to create container and cause the issue**
```
Run-ALValidation -app ./Foo.app -previousApps ...etc...
```

**Full output of scripts**
```
Unexpected error while validating app. Error is: The variable '$binaryReader' cannot be retrieved because it has not been set.
#Note: This is from the finally block in Extract-AppFileToFolder
```

**Comment by freddydk** [comment]
What version are you running - I think I fixed this?

**Comment by jhoek** [comment]
1.0.17. I think I ran Update-Module earlier today.

**Comment by freddydk** [comment]
I removed my comment, you are right.

**Comment by freddydk** [comment]
Shipped in 1.0.18
